draft-ietf-masque-connect-udp-listen.md

```diff
@@ -532,19 +540,22 @@ proxied over HTTP back to the client.
                         Context ID = 4
                         UDP Payload = Encapsulated UDP Payload
 
-/* Request packets without a corresponding context to be dropped*/
+/* Request packets without a corresponding compressed Context */
+/* to be dropped by closing the uncompressed Context */
  CAPSULE                       -------->
    Type = COMPRESSION_CLOSE
    Context ID = 2
 
 
-
 /* Proxy confirms unmapped IP rejection. */
             <-------- CAPSULE
                         Type = COMPRESSION_CLOSE
                         Context ID = 2
-/* Proxy drops any packets received on the
-bound IP(s):Port */
+
+/* All traffic that does not belong to the registered */
+/* Context ID 4 = 223.0.2.31:4321 is dropped at the proxy */
```

Comment:
Wait context ID 4 should still be open here right?

Reply:
I updated it to make a bit more sense